terragrunt/modules/crates-io/cloudfront-functions/static-router.js

```diff
@@ -19,3 +25,13 @@ function handler(event) {
 
     return request;
 }
+
+function permanentRedirect(destination) {
+    return {
+        statusCode: 301,
+        statusDescription: 'Moved Permanently',
+        headers: {
+            'location': { value: destination },
+        },
+    };
+}
```

Comment:
found the syntax here: https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/functions-tutorial.html